packages/react/src/hooks/useIsomorphicLayoutEffect.ts

```diff
@@ -0,0 +1,8 @@
+// use-isomorphic-layout-effect.js
```

Comment:
I think this comment can be removed. I might've accidentally left it here?

Reply:
- [x] remove obsolete comment